scss/inc/_forms.scss

```diff
@@ -342,7 +342,7 @@ input[type="password"].viewable-hiddenbox-input {
         }
     }
 
-    + .form-error {
+    & + .form-error {
         color: $error;
         text-align: right;
     }
```

Comment:
I think we can add such rules for cover select2 elements
```
& + .select2-choices {
  border-color: $error;
  + .select2-input {
       color: $error;
   }
}
```

Reply:
In order to avoid to many nested rules and still support `+ .form-error {` that it works, I push another solution :)